### DISCUSSION TITLE
Is there a plan to move to native fetch api on node 18?

### DISCUSSION THREAD

**Discussion by MiltonAkash** [sindresorhus/got#2242] [General]
Since fetch has become available natively from node 18? Is there a plan to use this and reduce the package size of this????

**Comment by sindresorhus** [comment]
No plans. Fetch is a very limited API and not even half of what Got does would be possible if we used Fetch. Check out [Ky](https://github.com/sindresorhus/ky) which is by the same authors and based on Fetch.

**Reply by MiltonAkash** [reply]
I'm already using Ky on client side. needed one equivalent but less package size for server side. Felt if something uses native fetch the package size will be less and more efficient.

**Reply by sindresorhus** [reply]
Ky works on Node.js. If you're on Node.js 18, it has native Fetch. If not, you can use https://github.com/sindresorhus/ky-universal
